substrate/validator-sets/pallet/src/lib.rs

```diff
@@ -64,18 +65,11 @@ pub mod pallet {
 
   /// The current session for a network.
   ///
```

Comment:
This not being `//` may add a new line to the doc comments which shouldn't be there. I'm unsure if it'll trim...